server/auth/auth.go

```diff
@@ -683,7 +683,28 @@ func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) err
 	// should be appropriate checks on order submission and match/swap
 	// initiation (TODO).
 
-	// TODO: notify client of penalty / account status change?
+	// Notify user of penalty.
+	details := "You may no longer trade. Leave your client running to complete outstanding trades."
+	penalty := &msgjson.Penalty{
+		Rule:     dex.Bytes([]byte{byte(rule)}),
```

Comment:
spec says this is a single byte, so why not use `byte` or `uint8` for this?  `dex.Bytes` is all kinds of weird especially in `(*Penalty) Serialize()`